SpatialGDK/Source/SpatialGDK/Private/SpatialView/EntityQuery.cpp

```diff
@@ -0,0 +1,118 @@
+﻿#include "SpatialView/EntityQuery.h"
+
+namespace SpatialGDK
+{
+
+EntityQuery::EntityQuery(const Worker_EntityQuery& Query)
+	: ResultType(static_cast<Worker_ResultType>(Query.result_type))
```

Comment:
Stupid question, but what's the benefit of storing this as a custom EntityQuery, over just keeping it as a Worker_EntityQuery?

Reply:
Worker_EntityQuery doesn't own anything so can't be moved around. Either everything it points to is in local scope or it's stored somewhere else and you need to know to delete it all once you are done.
This is just a wrapper to do that latter.